pyproject.toml

```diff
@@ -65,6 +65,23 @@ build-backend = "poetry.core.masonry.api"
 formats = "ipynb,py:percent"
 
 
+[tool.ruff]
+line-length = 120
+
+
+[tool.ruff.lint]
+select = ["ALL"]
+ignore = ["D", "FA", "PLR", "S", "PD", "SLF", "PT", "FBT", "C", "ANN", "EM", "TRY", "ARG", "FIX", "TD", "TID", "ERA", "INP", "E", "SIM", "RUF", "RET"]
+
+
+[tool.ruff.lint.isort]
+known-first-party = ["hamilflow"]
+
+
+[tool.black]
+line-length = 120
```

Comment:
Oh this is too long.

Reply:
Unless there is a very strong argument for 120, I would vote for black default line length:

https://test-black.readthedocs.io/en/latest/the_black_code_style.html#line-length

Reply:
21d5c96f33624883836a31f03973d391b216a67a